package.json

```diff
@@ -9,15 +9,29 @@
   },
   "license": "MIT",
   "dependencies": {
-    "express": "~4.11.x",
-    "kerberos": "~0.0.x",
-    "parse": "~1.8.0",
-    "parse-server": "*"
+    "express": "4.17.1",
+    "kerberos": "1.1.4",
+    "parse": "2.19.0",
+    "parse-server": "4.5.0"
   },
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.js",
+    "lint": "eslint --cache ./cloud && eslint --cache index.js",
+    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js",
+    "test": "jasmine"
   },
   "engines": {
     "node": ">=4.3"
+  },
+  "devDependencies": {
+    "eslint": "^7.17.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "babel-eslint": "^10.1.0",
+    "jasmine": "^3.6.3",
+    "mongodb-runner": "^4.8.0",
+    "bluebird": "^3.7.2"
```

Comment:
@dblythy I am wondering now why this repo suddenly depends on `bluebird`? Is this really necessary?

Reply:
Yes, it is used in the [test runner](https://github.com/parse-community/parse-server-example/blob/master/spec/utils/test-runner.js#L1).

Note that this from a different repo: [Parse Test Runner](https://github.com/AmpMe/parse-server-test-runner/blob/master/src/index.js)

Reply:
We are also not using bluebird in parse-server, so I wonder why we need it here?

Reply:
I'm not sure to be honest. I just used the test runner that had already been created. I can work on refactoring.

Reply:
Well, maybe the way parse-server runs its tests is more up-to-date. The code referenced is from 2017.

Reply:
Yes, I agree. I'll work on it.

Reply:
Cool! I added #390.